### PR TITLE
shadowvpn 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by mikemcquaid**
Re-add shadowvpn based on the existing 0.2.0 tag.

CC @DomT4

### CODE REVIEW

**Comment by DomT4** [comment]
```
* libtool dependency should be
  depends_on "libtool" => :build
```

:trollface: 

I'm cool with this though. The git history lines up and the tag is in the same place as the former upstream's.

File: Formula/shadowvpn.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  end
+
+  def caveats; <<-EOS.undent
+    You will also need to correct the tun references in #{etc}/shadowvpn/*.conf
+    to fit the tun/tap interface on your system.
+    EOS
+  end
+
+  test do
+    output = shell_output("#{bin}/shadowvpn -h; true")
```
Comment (bfontaine) [inline Formula/shadowvpn.rb]: We should remove this `; true`.

Comment (DomT4) [inline Formula/shadowvpn.rb]: We might be able to get away with using `1`. It exits hard which was the problem, as far as I remember.

Comment (bfontaine) [inline Formula/shadowvpn.rb]: Yep, I meant we should use `shell_output("...", 1)` instead of `shell_output("... ; true")`.

Comment (DomT4) [inline Formula/shadowvpn.rb]: :+1:

**Pull request closed without merging**